worker/bench/graph_test.go

```diff
@@ -0,0 +1,68 @@
+package bench
+
+import (
+	"os"
+	"testing"
+)
+
+// TODO: make this a runnable example?
```

Comment:
This isn't a test, it generates the same output as the query and saves that as a file in order to iterate on the graph settings.